app/services/submissions/search_service.rb

```diff
@@ -20,6 +20,16 @@ def call
 
   private
 
+    def sanitize_query(query)
+      return nil if query.blank?
+
+      query
+        .gsub(/[()]/, "")
+        .gsub(/[{}]/, "")
+        .gsub(/[\[\]]/, "")
```

Comment:
Another thing that might bite us in the back if we go down the route of simply removing the character is it will change the input search term. For example if someone types in "Chester&Noble", the search term will be "ChesterNoble" - whereas, it would yield better results as "Chester Noble" where the search terms are separate

Reply:
Ampersands aren't being removed, and the search is using partial matching anyway so it would still find it. See other comment anyway, this is redundant now.